app/views/govuk_publishing_components/components/_devolved_nations.html.erb

```diff
@@ -0,0 +1,23 @@
+<%
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+  devolved_nations_helper = GovukPublishingComponents::Presenters::DevolvedNationsHelper.new(local_assigns)
+
+  applies_to ||= t('components.devolved_nations.applies_to')
+  heading_level ||= 2
+%>
+
+<% if national_applicability.any? { |k,v| v[:applicable] == true } %>
+  <%= tag.section class: "gem-c-devolved-nations" do %>
+    <%= content_tag(shared_helper.get_heading_level, class: "govuk-heading-s govuk-!-margin-bottom-0") do %>
```

Comment:
You should probably also have a test that the component will render different heading levels when passed. I've just checked, and other components that use this shared helper have such tests.